### PR TITLE
COMMERCE-11660 | UI to show Price of SKU Per Unit of Measure

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
https://liferay.atlassian.net/browse/COMMERCE-11660

Resent from #4183

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -0,0 +1,312 @@
+@component-name = "portal-commerce"
+definition {
+
+	property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false${line.separator}feature.flag.COMMERCE-11287=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Product Info Management";
+
+	setUp {
+		CommerceConfiguration.commerceSetUp();
```
[the remark this answers is not in the document]
Comment (22LucaGiannotta22) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:10]: In this .testcase there will be tests without Minium not merged yet

**Comment by 22LucaGiannotta22** [comment]
relevant failures are not related to the PR. It can be force forwarded

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:stop

**Comment by 22LucaGiannotta22** [comment]
ci:stop

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
Relevant failures are not related to this PR. It can be force forwarded

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**